src/overlays/actors/ovl_En_Test4/z_en_test4.c

```diff
@@ -33,253 +35,288 @@ ActorInit En_Test4_InitVars = {
 };
 
 static s32 sIsLoaded = false;
+
 // "Night of ..."
-static s16 sNightMessages1[] = { 0x1BB4, 0x1BB5, 0x1BB6 };
+static s16 sNightOfTextIds2[] = { 0x1BB4, 0x1BB5, 0x1BB6 };
 // "Dawn of ..." (Note: first two message are the same)
-static s16 sDayMessages1[] = { 0x1BB2, 0x1BB2, 0x1BB3 };
+static s16 sDawnOfTextIds2[] = { 0x1BB2, 0x1BB2, 0x1BB3 };
 // "Night of ..."
-static s16 sNightMessages2[] = { 0x1BB4, 0x1BB5, 0x1BB6 };
+static s16 sNightOfTextIds[] = { 0x1BB4, 0x1BB5, 0x1BB6 };
 // "Dawn of ..." (Note: first two message are the same)
-static s16 sDayMessages2[] = { 0x1BB2, 0x1BB2, 0x1BB3 };
-static u16 D_80A43364[] = { CLOCK_TIME(6, 0), CLOCK_TIME(18, 0) };
+static s16 sDawnOfTextIds[] = { 0x1BB2, 0x1BB2, 0x1BB3 };
```

Comment:
If I'm understanding it right, the "2" variants are used with `EnTest4_HandleDayNightSwapFromInit`, and the other variants are used with `EnTest4_HandleDayNightSwap`. Wouldn't it be cleaner to just have them be in-function static? That way, you could just use the name name in both functions.

Reply:
Yeah, that makes more sense actually